.github/MAP_GUIDELINES.md

```diff
@@ -54,3 +54,22 @@ If these features do not work in the current codebase then your map PR will be *
 - Feel free to take inspiration from other maps, but please don't copy paste large parts of them.
 - If you use random item spawners, try using the types of them that create specific numbers of items. This way you don't risk accidentally overloading rooms with items because RNG decided it to be so.
 - Spacemas ornaments are now automatically generated as appropriate. You must only manually add the tree somewhere, specifically the ephemeral subtype.
+
+## Less critical but still important
+- **Place Directions!** They make new maps a lot easier to navigate, not just for players, but for you as well during the process. Use the donut 2/nadir floor decals, or the wall signs like cogmap 1, or even both. The amount of directions you should place should be proportional to map size, hallway size, and mazelikeness.
+- **Make departments easily recognisable!** Use unique floor designs, colored tiles, lines, and "doormats" to highlight to players where they are. Try to avoid big plain floors: spice things up with some checkered patterns or symbols. Different departments have different colours that they generally go by, so try to stick to those (blue for medbay (with black accents in robotics, and purple in pharmacy), purple for science and research, yellow for engineering, mining and mechanics and so on). Inside departments, use unique floor designs, coloured tiles, lines, and "doormats" (little spots of a department's colour outside their door). It just makes everything look nicer and more organised.
+- Break up empty hallways with things that seem realistic to have on that station. Donut2 is kind of old, so it has random tables and chairs, food and writing things. Things that make it feel lived in, stuff that adds flavour.
+- **Keep some crime in mind!** Maybe make some areas with less camera coverage, back rooms that are a little odd and musty. Give antags places to play with too.
```

Comment:
"Maybe make some areas, especially maintenance, with less camera coverage"
Maintenance should, imo, have very sparse camera coverage.

Reply:
gotcha